src/selectors/data/venuesSelectors.js

```diff
@@ -0,0 +1,37 @@
+import createCachedSelector from 're-reselect'
+
+export const selectVenues = state => state.data.venues || []
+
+export const selectVenuesByOffererIdAndOfferType = createCachedSelector(
+  selectVenues,
+  (state, optionalOffererId) => optionalOffererId,
+  (state, optionalOffererId, optionalOfferType) => optionalOfferType,
+  (venues, optionalOffererId, optionalOfferType) => {
+    let filteredVenues = venues
+
+    if (optionalOffererId)
+      filteredVenues = filteredVenues.filter(v => v.managingOffererId === optionalOffererId)
+
+    if (optionalOfferType) {
+      if (optionalOfferType.offlineOnly) filteredVenues = filteredVenues.filter(v => !v.isVirtual)
+      else if (optionalOfferType.onlineOnly)
+        filteredVenues = filteredVenues.filter(v => v.isVirtual)
+    }
+
+    return filteredVenues
+  }
+)(
+  (state, optionalOffererId, optionalOfferType) =>
+    `${optionalOffererId || ''}/${optionalOfferType || ''}`
+)
+
+export const selectPhysicalVenuesByOffererId = createCachedSelector(
```

Comment:
```suggestion
export const selectPhysicalVenuesByOffererIdAndOfferType = createCachedSelector(
```
C'est le moins pire mais à rechallenger car pas terrible non plus.

Reply:
En fait, non. Parce que `selectPhysicalVenuesByOffererId` renvoie par defaut les _Physical_ venues. C'est a dire celles qu'il ont pour offerType != isVirtual. 
Donc la notion d'offerType est bien deja inclue dans le nom de la methode (`Physical`)